README.md

```diff
@@ -95,6 +95,13 @@ During synthesis, the tags will be created using the attribute name as key and i
 
 If some of the tags' attributes are not present on the telemetry message received, the entity will still be synthesized with the available tags (if any).
 
+```yaml
+  tags:
+    - attribute: attributeNameB
```

Comment:
Is is worth adding the attribute keyword? if we just leave `attributeNameB` it would also work, but being more concise?

Reply:
its an object so i think it needs it

Reply:
This is how we do it in golden metrics:

https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/definitions/infra-awsapigatewayapi/golden_metrics.yml#L9

Each of these are objects, but the name of the metric functions as identifier:

![image](https://user-images.githubusercontent.com/346110/107408335-1ec56f80-6b0b-11eb-9ee8-37b370f1833e.png)

Reply:
Personally, I prefer it the way it is. From a clients perspective I don't think either way has much impact.
from an internal perspective, IMO it it is easier to operate on in code like such:
```
class Tag(attribute, multiValue = true)
List<Tag>
```
as apposed to 
```
class TagProperties(multiValue=true)
List<Map<String, TagProperties>
```

Reply:
@jfjoly any preference?